dev-db/redis/redis-5.0.8.ebuild

```diff
@@ -0,0 +1,160 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools flag-o-matic systemd toolchain-funcs
+
+DESCRIPTION="A persistent caching system, key-value and data structures database"
+HOMEPAGE="https://redis.io"
+SRC_URI="http://download.redis.io/releases/${P}.tar.gz"
```

Comment:
This has https:
https://github.com/antirez/redis/archive/5.0.8.tar.gz

Seen on https://redis.io/download , look for unstable ...

Reply:
If possible I'll rather use the official upstream url.